store_test.go

```diff
@@ -371,7 +372,12 @@ func TestLimitStore(t *testing.T) {
 	traces, _ := ms.Traces()
 	want := []*Trace{
 		{Span: Span{ID: SpanID{2, 3, 4}}},
-		{Span: Span{ID: SpanID{3, 4, 5}}},
+		{
+			Span: Span{ID: SpanID{3, 5, 6}},
+			Sub: []*Trace{
+				{Span: Span{ID: SpanID{3, 4, 5}}},
```

Comment:
Should the subspan trace be `3, 5, 6` and the containing span be `3, 4, 5`? Not sure what the semantics are given the ordering of the calls to `Collect()`, but just want to double check.